tests/testthat/test-bind_tables.R

```diff
@@ -0,0 +1,49 @@
+# Load initial variables
+token <- Sys.getenv("REDCAPTIDIER_LONGITUDINAL_API")
+redcap_uri <- Sys.getenv("REDCAP_URI")
+
+test_that("bind_tables works with no specifications", {
+
+  read_redcap_tidy(redcap_uri, token) %>%
+    bind_tables()
+
+  expect_true(exists("nonrepeated", envir = global_env()))
+  expect_true(exists("nonrepeated2", envir = global_env()))
+  expect_true(exists("repeated", envir = global_env()))
+})
+
+test_that("bind_tables works with environment specification", {
+
+  sample_env <- new_environment()
+
+  read_redcap_tidy(redcap_uri, token) %>%
+    bind_tables(environment = sample_env)
+
+  expect_true(exists("nonrepeated", envir = sample_env))
+  expect_true(exists("nonrepeated2", envir = sample_env))
+  expect_true(exists("repeated", envir = sample_env))
+})
+
+test_that("bind_tables works with forms specification", {
```

Comment:
The way this test is written, it tests for both the `environment` and `redcap_form_name` specification. Unit tests should really only test one behavior in isolation so I would use `global_env` not `sample_env` for this test.

Also, I would actually test whether you can pass `c("repeated", "nonrepeated")` to `redcap_form_name` to make sure passing multiple form names works as expected.

Reply:
I wrote the test to that it checks `c()` and it does still work 👍 

I'm having an issue with using `global_env()` instead of `sample_env()`, the tests wind up keeping the data objects between tests which results in failures. I looked into `withr::with_environment` but couldn't get it working, for now I added in a `rm()` line at the end of the tests to remove the expected objects but realize this is probably not good practice.

Reply:
I think your solution is fine... nit: would add a comment before the `rm()` line, like so:
```
# Clean up
rm(...)
```
Also, I don't think you need to specify `global_env()`, do you?

Reply:
Just tried it on my home Windows machine, and had this happen on the Mac as well. Without specifying the `global_env()`, objects still get loaded into it and the tests wind up failing. I can get around this by adding `envir = global_env()` to the `rm()` statements but then it's just a preference of where you want it specified. Unsure why this happens, since the default is to use the "active" directory and `identical(global_env(), current_env())` is `TRUE`.

Maybe it has to do with the `testthat` environment, but I don't really know.